q2_fragment_insertion/_format.py

```diff
@@ -19,3 +20,26 @@ def sniff(self):
 
 PlacementsDirFmt = model.SingleFileDirectoryFormat(
     'PlacementsDirFmt', 'placements.json', PlacementsFormat)
+
+
+class RAxMLinfoFormat(model.TextFileFormat):
```

Comment:
I think a better way is to name this format as `ModelParameters` format. Because in pplacer, it allows you to supply either a RAxML's info file, or a PhyML's stats file (see pplacer's parameter `-s`).